### PR TITLE
add kind to quickstart

### PR DESCRIPTION
**Pull request by Nikita-Levuskin**
### Changelog entry 

...

### Changelog category 

* Improvement

### Additional information

...

### CODE REVIEW

File: ydb/docs/ru/core/quickstart.md [hunk 1]
```diff
@@ -289,6 +333,32 @@ FLATTEN LIST BY keys AS key
...
 
+- Kind
+
+   Чтобы удалить базу данных {{ ydb-short-name }}, достаточно удалить сопоставленный с ней ресурс Database:
+
+   ```bash
+   kubectl delete database.ydb.tech database-kind-sample
+   ```
+
+   Чтобы удалить кластер{{ ydb-short-name }}, выполните следующие команды (все данные будут потеряны):
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/quickstart.md:344]:
```suggestion
   Чтобы удалить кластер {{ ydb-short-name }}, выполните следующие команды (все данные будут потеряны):
```

File: ydb/docs/en/core/quickstart.md [hunk 2]
```diff
@@ -135,6 +135,50 @@ Normally, {{ ydb-short-name }} stores data on multiple SSD/NVMe or HDD raw disk
...
+      kind create cluster --config=samples/kind/kind-config.yaml  --wait 5m
+      ```
+
+   5. Install the {{ ydb-short-name }} controller in the cluster:
+
+      ```bash
+      helm upgrade --install ydb-operator deploy/ydb-operator --set metrics.enabled=false
+      ```
+
+   6. Apply the manifest for creating a {{ ydb-short-name }} cluster:
```
Comment (vgvoleg) [inline ydb/docs/en/core/quickstart.md:162]: is it accurate to call a `Storage` object as a `YDB cluster`?

File: ydb/docs/en/core/quickstart.md [hunk 3]
```diff
@@ -135,6 +135,50 @@ Normally, {{ ydb-short-name }} stores data on multiple SSD/NVMe or HDD raw disk
...
+
+   2. Install [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+   3. Clone the repository with [{{ ydb-short-name }} Kubernetes Operator](https://github.com/ydb-platform/ydb-kubernetes-operator):
+
+      ```bash
+      git clone https://github.com/ydb-platform/ydb-kubernetes-operator && cd ydb-kubernetes-operator
+      ```
+
+   4. Run `Kind` with `kind-config.yaml`:
```
Comment (blinkov) [inline ydb/docs/en/core/quickstart.md:150]:
```suggestion
   4. Create a Kind cluster and wait until it is ready:
```

File: ydb/docs/ru/core/quickstart.md [hunk 4]
```diff
@@ -289,6 +333,32 @@ FLATTEN LIST BY keys AS key
...
+   kubectl delete storage.ydb.tech storage-kind-sample
+   ```
+
+   Чтобы удалить контроллер {{ ydb-short-name }} из кластера Kubernetes, удалите релиз, созданный Helm:
+
+   ```bash
+   helm delete ydb-operator
+   ```
+
+   Чтобы удалить кластер `kind` целиком, выполните следующую команду:
```
Comment (blinkov) [inline ydb/docs/ru/core/quickstart.md:356]:
```suggestion
   Чтобы удалить кластер Kind целиком, выполните следующую команду:
```

File: ydb/docs/ru/core/quickstart.md [hunk 5]
```diff
@@ -134,6 +134,50 @@
...
+
+   2. Установите [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+   3. Склонируйте репозиторий с [{{ ydb-short-name }} Kubernetes Operator](https://github.com/ydb-platform/ydb-kubernetes-operator):
+
+      ```bash
+      git clone https://github.com/ydb-platform/ydb-kubernetes-operator && cd ydb-kubernetes-operator
+      ```
+
+   4. Запустите `Kind` с использованием `kind-config.yaml`:
```
Comment (blinkov) [inline ydb/docs/ru/core/quickstart.md:149]:
```suggestion
   4. Создайте Kind кластер и дождитесь его готовности:
```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**